polar-core/src/debugger.rs

```diff
@@ -325,7 +324,7 @@ impl Debugger {
                     let mut vars = vm
                         .bindings(true)
                         .keys()
-                        .map(|k| k.to_polar())
+                        .map(|k| k.0.clone())
```

Comment:
Tiny nit, but do we need this line? It seems like rust should let us call `join()` somehow without explicitly cloning each element of the list.

Reply:
It was just me being lazy with the borrow checker since it wasn't happy with my original attempt of `.map(|k| &k.0)`. Updated to use `AsRef` in a153f52b3c549d8373bb26702f16f6f9b5056b83